.gitignore

```diff
@@ -20,6 +20,7 @@ target/
 test-output/
 APICalls.*
 Rest.*
+src/test/resources/Features/OCD-2006-UploadListingsRegularly.feature
```

Comment:
Why ignore this file?

Reply:
It's not complete yet and the steps were not implemented, not ready for review.

Reply:
I am going to take it out in next commit.